feature-libs/quote/components/details/cart/quote-details-cart.component.spec.ts

```diff
@@ -82,6 +103,94 @@ describe('QuoteDetailsCartComponent', () => {
     expect(component).toBeTruthy();
   });
 
+  describe('Ghost animation', () => {
+    it('should render view for ghost animation', () => {
+      component.quoteDetails$ = of(null);
```

Comment:
strict mode issue - null is not valid here

Reply:
done